gluefactory/models/base_model.py

```diff
@@ -60,6 +61,8 @@ class BaseModel(nn.Module, metaclass=MetaModel):
     required_data_keys = []
     strict_conf = False
 
+    weights_initialized = False
```

Comment:
```suggestion
    are_weights_initialized = False
```
and rename all other mentions